pkg/cmd/build/build.go

```diff
@@ -18,6 +18,7 @@ import (
 	"os"
 	"path/filepath"
 
+	oktetoErrors "github.com/okteto/okteto/pkg/errors"
```

Comment:
we use `okErrors` in other parts of the repo. 

```suggestion
	okErrors "github.com/okteto/okteto/pkg/errors"
```